src/faq.html

```diff
@@ -74,7 +74,27 @@
           <li>Make sure the device is in pairing mode. To find out how to get your device into pairing mode, check the device documentation. For some devices, pairing mode instructions are also documented on <a href="https://zigbee.blakadder.com">zigbee.blakadder.com</a.</li>
           <li>Check the <a href="https://www.home-assistant.io/integrations/zha/#best-practices-to-avoid-pairingconnection-difficulties">Best practices to avoid pairing issues with Zigbee devices</a>.</li>
         </ol>
-      </p>        
+      </p>
+  - title: >
+      Some of my ZHA devices are now shown as unavailable or suddenly stop working.
+    content: >
+      <p>You can try the following steps:
+        <ol>
+          <li>Refresh your browser page (Ctrl+Shift+R).</li>
+          <li>Power cycle the device.</li>
```

Comment:
One common situation people find themselves in is when they join a new device near the coordinator, then move the device to the other end of their home. Many devices will maintain the weak connection to their original router and work unreliably.

Here's one possible wording:
```suggestion
          <li>Power cycle the device.</li>
          <li>Ensure you added the device to your network in its final position in your home and not just near the coordinator. Many devices will maintain a weak connection to the router they first joined through instead of picking a new one nearby.</li>
```

Reply:
thank you for your input, @puddly :pray: